admin-src/utils/index.js

```diff
@@ -20,3 +20,18 @@ export const getDownloadURLFromImages = async (imageReferences) => (
     Promise.resolve([])
   )
 )
+
+export const userHasAccess = async () => {
+  try {
+    const user = await firebase.auth().currentUser
+
+    if (!user) return false
+
+    const result = await user.getIdTokenResult()
+
+    return Boolean(result.claims.isAdmin)
+  } catch (error) {
+    console.error(error.message)
```

Comment:
I do not think this will work as expected. I'm not quite sure because this is the web environment, but in mobile a `console.error` will stop the execution of the app, so the `return false` will never be reached.

![image](https://user-images.githubusercontent.com/35865469/70820785-41011780-1db8-11ea-9cf6-21ecbaba533f.png)


It probably never happened because `getIdTokenResult` did not fail. I suggest faking an error throw to see if this works as expected. I ping Nahuel for web specific insight

@nahudalla